seqr/views/apis/data_manager_api.py

```diff
@@ -317,8 +317,8 @@ def _get_sample_file_path(file_dir, sample_guid):
 
 @pm_or_data_manager_required
 def load_rna_seq_sample_data(request, sample_guid):
-    sample = Sample.objects.get(guid=sample_guid)
-    logger.info(f'Loading outlier data for {sample.sample_id}', request.user)
+    sample = RnaSample.objects.get(guid=sample_guid)
+    logger.info(f'Loading outlier data for {sample.individual.individual_id}', request.user)
 
     request_json = json.loads(request.body)
     file_name = request_json['fileName']
```

Comment:
(Unrelated to this pr, but I'm poking around in here.).  Looks like `model_cls.bulk_create` below (line 339) is missing the `batch_size` arg that exists in `management/commands/load_rna_seq.py`.  Should we add it?

Reply:
I would say if you are making a PR to change the batch sizefor a bulk create, it would be in-scope to add it there